test/utils.spec.js

```diff
@@ -0,0 +1,30 @@
+const utils = require('../src/utils');
+
+describe('when validating component', () => {
+  describe('when component\'s handlebars version is obsolete', () => {
+    const template = {compiler:[6,'>= 2.0.0-beta.1'],main:function(){return'Hello world!';},useData:!0}; // eslint-disable-line
+    const result = utils.validator(template, 'handlebars');
+
+    test('should error', () => {
+      expect(result.isValid).toBe(false);
+      expect(result.error).toBe('The component can\'t be rendered because it was published with an older OC version');
+    });
+  });
+
+  describe('when component\'s handlebars version is supported', () => {
+    const template = {compiler:[7,'>= 4.0.0'],main:function(){return'Hello world!';},useData:!0}; // eslint-disable-line
+    const result = utils.validator(template, 'handlebars');
+
+    test('should be valid', () => {
+      expect(result.isValid).toBe(true);
+    });
+  });
+
+  describe('when component\'s template type is jade', () => {
+    const result = utils.validator('', 'jade');
```

Comment:
Why the utils.validate signature has even the type param? This module should just be about handlebars?